context/app/static/js/components/home/Hero/const.tsx

```diff
@@ -11,7 +11,7 @@ export const HOME_TIMELINE_ITEMS: TimelineData[] = [
     titleHref: '/cells',
     description: 'Search datasets by cell type name or Cell Ontology ID.',
     date: 'May 2024',
-    img: <entityIconMap.CellType />,
+    img: <entityIconMap.CellType fontSize="1.5rem" />,
```

Comment:
Pull out the fontSize for reuse?

Reply:
I created a `timelineIconProps` object which we can pass any additional props into in the future 👍🏻